lib/fauna/query.rb

```diff
@@ -300,6 +311,17 @@ def self.divide(*numbers)
 
   private
 
+    # Queries that accept lambdas can take either a manually construced lambda or a block.
+    def self.to_lambda(lambda_expr, lambda_block)
```

Comment:
I don't know that we need this method? Couldn't we just do something like `expr || lambda(&block)` inline? Or `expr = lambda(&block) if block`, if you want it to prefer the block over expr